### PR TITLE
Workspace from URL

### PR DESCRIPTION
**Pull request by drg101**
Allows the workspace to be seeded by the URL.

To test, you can add this string to your URL in the browser and reload:
`?workspace=N4IgNghgngpgTgZxALgNqgMYHsxhhgFwEssA7FEAuCQgfQFsYATIiU2gCywFcEYuwTWkVLZGIADQgyKKtxhSYADwAObJswDKBCARgoAZhDB8p2UgioQRBbbv1oQAJgAMARhcNmrdl178cIRExfQBdMzJLahskNFCAXwlMHDxCEnJkEGxuUgIoLxY2Th4+ASDRLHEpGWQ5BRBlNVINJjs9WTh5CIsrGzaHVGd3T0ZC3xKAwWEK8XCsyN7c2PQQUghxTNcPAp9i-zLpkMkQS3sUVCcAFgBOa4kAVgA2W+uEhKT53HxiGpOANyItGyuXyAHEAJKaABSAHlwQA5Y41OqKVTqLQ6drIIwmermKLWXL9c4gABKAAUADK0AAqAAkAKIAWQZmhAc3xiwIy3eyS+aV+CABtCsdAh0LhiOqGRRDTRzQxZ1qnTxC2iRMxAzJVNpjJZbI5asJ3POoDWG211PpzNZx1OWNQLgkLgAdABme5veJzGzwPiEDryeJAA`

This should seed your menu with this (it may take awhile to load because it has to wait for all the options in the menu to be loaded before it loads in): image

### CODE REVIEW

**Comment by drg101** [comment]
@Marc-9 Could you set it up so the "workspace" URL param gets forwarded on the live site to the iFrame. This will allow us to seed aperture with a specific workspace on the live-site by modifying the `workspace` parameter of: `https://urban-sustain.org/services/aperture.php?workspace=<workspaceString>`.

**Comment by Marc-9** [comment]
> @Marc-9 Could you set it up so the "workspace" URL param gets forwarded on the live site to the iFrame. This will allow us to seed aperture with a specific workspace on the live-site by modifying the `workspace` parameter of: `https://urban-sustain.org/services/aperture.php?workspace=<workspaceString>`.

I can have this done in 3-5 business days, or you can request express shipment which will cost you an emoji

**Review by honeybadgerofdoom — APPROVED** [review]
Well this is super cool

File: src/js/ui/NewDataExploration/Workspace.js [hunk 1]
```diff
@@ -77,17 +77,29 @@ export function prettifyJSON(name) {
     return Util.capitalizeString(Util.underScoreToSpace(name));
 }
 
+let workspaceIsLoaded = false;
```
Comment (pierce-smith1) [inline src/js/ui/NewDataExploration/Workspace.js:80]: Having globals float around feels like bad practice, is there a better way to do this?

Comment (drg101) [reply src/js/ui/NewDataExploration/Workspace.js:80]: Yes, in state, I'll change it to this

**Review by pierce-smith1 — APPROVED** [review]
React continues to make less and less sense by the day...

**Pull request merged**